src/subsplit_dag_storage.hpp

```diff
@@ -181,6 +192,8 @@ class GenericNeighborsView {
     bool operator==(const Iterator& other) const { return i_ == other.i_; }
 
     VertexId operator*() { return i_->first; }
+
+    VertexId GetNodeId() const { return i_->first; }
```

Comment:
Ha! I clearly didn't look this over very closely on the first round. Having a member variable named `i_` clearly isn't enough to know what it is. `iter_`?